modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/AutocompleteMenuItem.ts

```diff
@@ -89,6 +89,7 @@ const renderAutocompleteItem = (
     onAction: (_api) => onItemValueHandler(spec.value, spec.meta),
     onSetup: Fun.constant(Fun.noop),
     triggersSubmenu: false,
+    // TINY-9125: Hmm. This is going to be a problem.
```

Comment:
Need to combine these somehow. Maybe fallback if it doesn't find anything in meta?